cirq/ops/common_channels.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Quantum channels that are commonly used in the literature."""
+
+from typing import Iterable
+
+import numpy as np
+
+from cirq import protocols
+from cirq.ops import raw_types
+
+
+class AsymmetricDepolarizingChannel(raw_types.Gate):
+    """A channel that depolarizes asymmetrically along different directions."""
+
+    def __init__(self, p_x: float, p_y: float, p_z: float) -> None:
+        """The asymmmetric depolarizing channel.
+
+        This channel applies one of the three Pauli operators at random,
```

Comment:
I didn't realize that these probabilities were describing disjoint events instead of independent events. Rewrite to make this clearer.

Reply:
Ha yeah that's sort of the definition :)  Rewrote to make this clear.